scalene/scalene_profiler.py

```diff
@@ -1562,15 +1502,18 @@ def main() -> None:
                         exit_status = 0
                         profiler.start()
                         # Run the code being profiled.
+                        import sys as _sys
+                        _sys.argv = ['abcde']
```

Comment:
What's this for?

Reply:
Debugging artifact from figuring out the behavior of `exec`